pkg/api/workflow/v1/types.go

```diff
@@ -24,6 +24,7 @@ import (
 )
 
 // +genclient
+// +genclient:noStatus
```

Comment:
I'm curious, what is the purpose of this tag?

Reply:
It removes `UpdateStatus` from client: see https://github.com/AmadeusITGroup/workflow-controller/pull/51/commits/c3e6d8789928e3f9bed44efdd707294cac2e0174#diff-dadd7163529d130c4dcc280d8b0d953bL86

Reply:
ok thanks